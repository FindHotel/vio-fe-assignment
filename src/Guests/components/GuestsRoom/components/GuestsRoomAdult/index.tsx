import React, { ReactElement } from "react";

import Button from "components/Button";
import { ReactComponent as IconPlus } from "assets/plus.svg";
import { ReactComponent as IconMinus } from "assets/minus.svg";

import { Guest, GuestText, GuestButtonGroup } from "../../styles";

const GuestsRoom = (): ReactElement => {
  return (
    <Guest>
      <GuestText>Adults</GuestText>
      <GuestButtonGroup>
        <Button theme="secondary">
          <IconMinus />
        </Button>
        <GuestText>0</GuestText>
        <Button theme="secondary">
          <IconPlus />
        </Button>
      </GuestButtonGroup>
    </Guest>
  );
};

export default GuestsRoom;
