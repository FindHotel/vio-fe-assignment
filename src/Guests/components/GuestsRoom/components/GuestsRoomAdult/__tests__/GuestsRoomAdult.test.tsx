import React from "react";
import { render } from "@testing-library/react";

import GuestsRoomAdult from "..";

describe("<GuestsRoomAdult />", () => {
  it("should render properly", () => {
    const { container } = render(<GuestsRoomAdult />);
    expect(container.firstChild).toMatchSnapshot();
  });
});
