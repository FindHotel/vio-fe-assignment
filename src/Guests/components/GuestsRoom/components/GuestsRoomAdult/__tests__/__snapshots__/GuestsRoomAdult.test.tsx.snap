// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<GuestsRoomAdult /> should render properly 1`] = `
<div
  class="sc-eCImvq guCNbL"
>
  <span
    class="sc-jRQAMF bgpgfc"
  >
    Adults
  </span>
  <div
    class="sc-gKckTs jduWvm"
  >
    <button
      class="sc-bdvvaa eWedmN"
    >
      <svg>
        minus.svg
      </svg>
    </button>
    <span
      class="sc-jRQAMF bgpgfc"
    >
      0
    </span>
    <button
      class="sc-bdvvaa eWedmN"
    >
      <svg>
        plus.svg
      </svg>
    </button>
  </div>
</div>
`;
